docs/runbooks/NoReadyVirtController.md

```diff
@@ -1,52 +1,64 @@
+<!-- Edited by Jiří Herrmann, 9 Nov 2022 -->
+
 # NoReadyVirtController
 
 ## Meaning
 
-From a high-level perspective the virt-controller is responsible for all the cluster wide virtualization functionality.
-
-This controller is responsible for monitoring the VMI (CRDs) and managing the associated pods. The controller will make sure to create and manage the life-cycle of the pods associated to the VMI objects.
+This alert fires when no available `virt-controller` devices have been detected for 5 minutes.
 
-A VMI object will always be associated to a pod during it's life-time, however, due to i.e. migration of a VMI the pod instance might change over time.
+A `virt-controller` handles monitoring the custom resource definitions (CRDs) of a virtual machine instance (VMI) and managing the associated pods. The device creates pods for VMIs and manages the lifecycle of the pods.
 
-This alert is triggered when no ready virt controllers were detected for the last 5 min.
+Therefore, `virt-controller` devices are critical for all cluster-wide virtualization functionality.
 
 ## Impact
-Complete failure in VM lifecycle management i.e. launching a new VM instance or shutting down an existing VM instance.
+Any actions related to VM lifecycle management fail. This notably includes launching a new VMI or shutting down an existing VMI.
 
 
 ## Diagnosis
 
-- Set the environment variable NAMESPACE
+1. Set the `NAMESPACE` environment variable:
+```bash
+$ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+```
+
+2. Verify a `virt-controller` device is available:
```

Comment:
I think "Verify the number of available `virt-controller` devices:" would be more accurate.